### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_Resources_ScreenReader: The word "Dialog " is not announced by JAWS.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#10645]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#NonBenchmark#DesktopApp;#MAS1.3.1;#SQL Azure Data Studio;#JAWS;#STP;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.3.1
Screen reader : JAWS

**Repro Steps:**
1 . Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 . Navigate till Resources section on Welcome screen.
4 . Launch screen reader as JAWS.
5 . Navigate to link documentation and hit enter.

**Actual:**
When the link documentation is activated and the dialog has opened the title and data are announced of the dialog but the word dialog of the control opened is not announced.
The same issue seen when get started is activated 

**Same issue not with NVDA**

**Same issue with the following screen:**
Videos pop-up

**Expected:**
The word dialog needs to be announced after the sentence "Azure data studio-insiders" followed by the rest of the data.

**User Impact**:
Screen reader users will not know a dialog has been opened.

**Attachment link for Reference:**
[10645A11y_AzureDataStudioStayHealthy_Welcome_Resources_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4705299/10645A11y_AzureDataStudioStayHealthy_Welcome_Resources_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
@fsteffi we tried this scenario with Narrator, and narrator does announce it as a dialog, looks like an issue with JAWS. I am closing the issue as external, please let me know if you have any questions.

also this dialog is a feature of vscode, you can open vscode and find an extension and click on the repository link on the extension page. if you think this is not a JAWS issue, please open an issue for vscode.

**Comment by fsteffi** [comment]
#closed;

on click on documentation now the screen is redirected to https://learn.microsoft.com/en-us/sql/azure-data-studio/?view=sql-server-ver16

Version: 1.43.0-insider (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/b4d5a08199e9e0cb0138ce540c086ac32d072019
Date: 2023-03-07T06:31:08.002Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300
